Slims/slims.py

```diff
@@ -0,0 +1,183 @@
+import os
+import re
+
+from slims.slims import Slims
+from slims.criteria import is_one_of, equals, conjunction, not_equals
+from slims.content import Status
+
+
+
+class slims:
```

Comment:
The naming of this variable could clash with the slims python api. Consider renaming it to something like `slims_credentials`.

Reply:
Since we're also dealing with credentials you probably ought to put them at the top level in a config and read from there.

Reply:
do you mean like in demuxer where you have a json file in /medstore/credentials that you open in the config?